portfolio/src/main/webapp/index.html

```diff
@@ -60,11 +60,13 @@ <h2>Comment Forum</h2>
 
         <label for="max-num-comments">Choose number of comments to display:</label>
         <input type="number" id="max-num-comments" value="5" onchange="displayCommentsToPage()">
+        <br/></br/>
```

Comment:
Consider using CSS margins or padding instead of line breaks. There's not really a technical difference but by convention HTML should be 'semantic', meaning it only contains content and metadata about that content, not anything related to styling.